eng/native/configurecompiler.cmake

```diff
@@ -620,6 +620,14 @@ if (CLR_CMAKE_HOST_UNIX)
 
     # clang 18.1 supressions
     add_compile_options(-Wno-switch-default)
+
+    # clang 20 suppressions
+    if (CMAKE_CXX_COMPILER_ID)
+      check_cxx_compiler_flag(-Wnontrivial-memaccess COMPILER_SUPPORTS_W_NONTRIVIAL_MEMACCESS)
```

Comment:
Do we need this check to make this conditional? We pass `-Wno-unknown-warning-option` above and we do not have this check for other warning options.

Reply:
I'll remove the check